src/index.js

```diff
@@ -75,5 +76,12 @@ module.exports = gapic.v1beta1;
  */
 module.exports.v1beta1 = gapic.v1beta1;
 
+/**
```

Comment:
With the release of v1, do we want the default to be v1? (line 70)

Reply:
Yes, absolutely!

Reply:
@crwilcox should `synthtool` take care of that as well?

Reply:
@alexander-fenster one of the next things we are adding is the ability to merge these index files. You can do it today with replacement patterns but it isn't elegant.